### DISCUSSION TITLE
Cant search image in sdwebui-reForge with this extension

### DISCUSSION THREAD

**Discussion by Moussack** [zanllp/infinite-image-browsing#707] [Q&A] [answered]
Hi i install this extension for sdwebui reForge and It seems i cannot search image in sdwebui reForge with this extension, 
The "generate index for search image" button don't do anything.

Is this sdwebui reForge specific issue or the extension??

**Comment by zanllp** [comment]
Please paste the displayed error message, which might appear in the terminal when starting Python or in the browser's console.

**Comment by Moussack** [comment]
There'e no error messages in the console, the button gives no response.
the only message when starting is "IIB Database file has been successfully backed up to the backup folder."

**Comment by zanllp** [comment]
Can you take a screenshot of the IIB emptystartpage, just like the one below?
![image](https://github.com/user-attachments/assets/0fe8f007-a0a4-4ef7-9e81-5f3487f1ce12)

**Comment by Moussack** [comment]
here it is

![chrome_Gzc9CI8uEF](https://github.com/user-attachments/assets/4463ba93-c7f7-46ff-b49e-99ce1121a523)

**Comment by zanllp** [comment]
This situation usually occurs when there are no images in any of the folders, or an error occurred during the first index generation. You can check the log.log file in the extension's root directory to see if there are any error records.

**Reply by Moussack** [reply]
images are exist in the folder and the log is empty, no errors are recorded

**Comment by zanllp** [comment]
I am currently unable to pinpoint this issue. I will add more logging outputs in a few days to help identify the problem. Previously, we tested it on sdwebui-forge and there were no issues.

**Reply by Moussack** [reply, marked as answer]
I'm using the sdwebui-**reForge**, not sdwebui-forge. Yes in sdwebui-forge the IIB is fully functional.
https://github.com/Panchovix/stable-diffusion-webui-reForge


Btw i've fixed it, i rebuild the index via the IIB's Global Setting. Now the search function is functional again, thx for the responses.

![NVIDIA_Share_HQ5B1OJEfI](https://github.com/user-attachments/assets/aea81eda-1fe5-46e9-a2e3-201d54c6fd09)
